docs/src/tutorials/tut_setup.md

```diff
@@ -1,13 +1,320 @@
+│ ══════════════════════════════                                                                   │
+│ #particles: ………………………………………………… 360                                                              │
+│ density calculator: …………………………… ContinuityDensity                                                │
+│ correction method: ……………………………… Nothing                                                          │
+│ state equation: ……………………………………… StateEquationCole                                                │
+│ smoothing kernel: ………………………………… SchoenbergCubicSplineKernel                                      │
+│ viscosity: …………………………………………………… ArtificialViscosityMonaghan{Float64}(0.02, 0.0, 0.01)            │
+│ density diffusion: ……………………………… nothing                                                          │
+│ acceleration: …………………………………………… [0.0, -9.81]                                                     │
+│ source terms: …………………………………………… Nothing                                                          │
+└──────────────────────────────────────────────────────────────────────────────────────────────────┘
+```
+
+## Boundary system
+
+In order to define a boundary system, we first have to choose a boundary model,
+which defines how the fluid interacts with boundary particles.
+We will use the [`BoundaryModelDummyParticles`](@ref) with
+[`AdamiPressureExtrapolation`](@ref), which generally produces the best results
+of the implemented methods.
+See [here](@ref boundary_models) for a comprehensive overview over boundary models.
+```jldoctest tut_setup; output = false
+boundary_model = BoundaryModelDummyParticles(tank.boundary.density, tank.boundary.mass,
+                                             state_equation=state_equation,
+                                             AdamiPressureExtrapolation(),
+                                             smoothing_kernel, smoothing_length)
+boundary_system = BoundarySPHSystem(tank.boundary, boundary_model)
+
+# output
+┌──────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ BoundarySPHSystem{2}                                                                             │
+│ ════════════════════                                                                             │
+│ #particles: ………………………………………………… 276                                                              │
+│ boundary model: ……………………………………… BoundaryModelDummyParticles(AdamiPressureExtrapolation, Nothing) │
+│ movement function: ……………………………… nothing                                                          │
+└──────────────────────────────────────────────────────────────────────────────────────────────────┘
+```
+
+## Semidiscretization
+
+The key component of every simulation is the [`Semidiscretization`](@ref),
+which couples all systems of the simulation.
+All methods in TrixiParticles.jl are semidiscretizations, which discretize
```

Comment:
`All methods`? That is a bit too general.

Reply:
Are they not? Do we have anything that is not a semidiscretization? Will we ever have a method that uses a space-time discretization? I've only seen this with grid-based methods.

Reply:
What you mean is "All simulation methods" or "All schemes"?